client/gutenberg/editor/calypsoify-iframe.tsx

```diff
@@ -859,6 +873,7 @@ const mapStateToProps = (
 		isSiteUnlaunched: isUnlaunchedSite( state, siteId ),
 		site: getSite( state, siteId ),
 		parentPostId,
+		mediaGutenframeActions: getGutenframeMediaActions( state, siteId ),
```

Comment:
Nit: does `gutenframeMediaActions` make more sense in terms of naming? It would also be consistent with the selector name.